src/components/Summary/index.ts

```diff
@@ -1,3 +1,2 @@
 export { default as AlterationSummary } from './AlterationSummary.vue'
 export { default as DocumentsDelivery } from './DocumentsDelivery.vue'
-export { default as NoFeeSummary } from './NoFeeSummary.vue'
```

Comment:
This was never used so I deleted it.

(It looks like it was left over from a previous thing.)